com.prusa3d.PrusaSlicer.appdata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
The desktop and appdata file should ideally be upstreamed.

Reply:
ideally yes I know, I am working on that. Until then can we keep it here? I will adjust as soon as it is upstream. Including the .desktop file

Reply:
@bilelmoussaoui  I opened a merge request to include these files, never the less if they do get accepted they wont be in until the next release.  
Please let me know if there is anything else needed. Thanks.